target/linux/ramips/image/rt305x.mk

```diff
@@ -389,6 +392,27 @@ define Device/dlink_dir-320-b1
 endef
 TARGET_DEVICES += dlink_dir-320-b1
 
+define Device/dlink_dir-506l
+  SOC := rt5350
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-506L
+  DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-spi-dev kmod-usb-serial \
+		kmod-usb-serial-option kmod-usb-net \
+		kmod-usb-net-cdc-ether comgt-ncm \
+		kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-ledtrig-timer
+  DLINK_ROM_ID := DLK6E2114001
+  DLINK_FAMILY_MEMBER := 0x6E21
+  DLINK_FIRMWARE_SIZE := 0x7E0000
+  KERNEL := $(KERNEL_DTB)
+  IMAGES += factory.bin
+  IMAGE/sysupgrade.bin := mkdlinkfw | pad-rootfs | append-metadata
+  IMAGE/factory.bin := mkdlinkfw | pad-rootfs | mkdlinkfw-factory
+  SUPPORTED_DEVICES += dir-506l
+  DEFAULT := n
```

Comment:
Why?

Reply:
I make all like in dlink_dwr-512-b section

Reply:
I see. `DEFAULT := n` means the device is not built by default, so this line tells the reviewer that there is no reason to merge your pull request, because it won't be built anyway.